### ISSUE TITLE
Hex colors with alfa channels are not updated (or with large delay) when changed by a formula in dashboard (overlay)

### ISSUE THREAD

**Issue by Shaftoe62** [SHWotever/SimHub#849]
When I use a formula to change a normal color it gets updated with a 'normal framerate'.

But when it is a color with transparency (#AACCCCCC to BBCCCCCC) nothing happens or only after a very long delay of (not sure) 30 seconds or so.

Or when I alt-tab out of the game.

**Comment by SHWotever** [comment]
Hi ! 30s is the safety delay when a formula error is triggered, you can find in the logs the errors, it's sometimes due to data not being availablebefore game startup, usually using isnull([xxxx], defaultvalue) allows to set a default value until live values are availalble .

**Comment by Shaftoe62** [comment]
Ok, I'll check the logs and let you know if it was a formula error or not. 

By now I've build in a workaround that works with the opacity itself but that requires an extra component and an extra binding. And that times 8. Much more expensive. So I hope I can get it to work.

Thanks

**Comment by Shaftoe62** [comment]
Ok, I found an error in yesterdays log but I dont think the error is in the (alfa) hex color format? The parser seem to reject that format...

[2021-06-15 08:55:16,107] ERROR - An error occured while executing binding on Gear-3-return gear_color(3)
color = '#7E000000'
if ($prop('GameRawData.Gear') == gear) {
	color = '#FF000000'
	if ($prop('CarSettings_RPMRedLineReached') == 1) {
		color = '#FFAA0000'
		}
	}
return color System.FormatException: Token is not valid.
   at MS.Internal.Parsers.ParseColor(String color, IFormatProvider formatProvider, ITypeDescriptorContext context)
   at System.Windows.Media.ColorConverter.ConvertFromString(String value)
   at SimHub.Plugins.OutputPlugins.GraphicalDash.EditorModel.Applybindings(IBindable item, String objectName, DashboardBindingData binding, IFormulaEngine GraphicalDashEngine), formula will not be evaluated for 30s

**Comment by Shaftoe62** [comment]
I wrote a quick testing function to test this fenomenon:

function opacity_test () {
	var base_color = 'ff0000'
	var rpm_percent = $prop('CarSettings_CurrentDisplayedRPMPercent')
	var alfa = clamp((Math.round(rpm_percent * 2.55)),0,255)
	return '#' + alfa.toString(16) + base_color
	}

function clamp(value, min, max) {
	return Math.min(Math.max(value, min), max);
	}


The return values look ok to me but I still get the same error:




&nbsp; | ERROR | An error occured while executing binding on RectangleItem1 return opacity_test() System.FormatException: Token is not valid. &nbsp;&nbsp;at MS.Internal.Parsers.ParseColor(String color, IFormatProvider formatProvider, ITypeDescriptorContext context) &nbsp;&nbsp;at System.Windows.Media.ColorConverter.ConvertFromString(String value) &nbsp;&nbsp;at SimHub.Plugins.OutputPlugins.GraphicalDash.EditorModel.Applybindings(IBindable item, String objectName, DashboardBindingData binding, IFormulaEngine GraphicalDashEngine), formula will not be evaluated for 30s
-- | -- | --

**Comment by SHWotever** [comment]
I loce as a duplicate of : https://github.com/SHWotever/SimHub/issues/849
